infrastructure/hub-and-spoke-peering/main.tf

```diff
@@ -273,7 +273,7 @@ module "vpn-spoke-2" {
   # routes exchanged via peering
   remote_ranges = ["10.0.0.0/8"]
   tunnels = {
-    spoke-2 = {
+    hub = {
```

Comment:
This is unrelated to the subnet change but I noticed that the tunnel in this example got created as "hub-hub". This fixes it.